producers-api/build.gradle

```diff
@@ -23,3 +23,7 @@ jar {
 bootJar {
     enabled = false
 }
+
+test {
+    useJUnitPlatform()
```

Comment:
do we have to do this in each module?

Reply:
As far as I'm aware. Would be nice for the change to be in the main build gradle but haven't got it to work.